distrho/DistrhoUtils.hpp

```diff
@@ -94,12 +94,12 @@ static inline
 void d_debug(const char* const fmt, ...) noexcept
 {
     try {
-        ::va_list args;
-        ::va_start(args, fmt);
+        va_list args;
```

Comment:
why is this specifically needed?

Reply:
It creates a syntax error at the `::` location.
Making a wild guess, `va` stuff might be macros on the windows platform.

Reply:
bummer but ok, lets go with removing :: then